repology/repoproc.py

```diff
@@ -309,7 +309,11 @@ def DeserializeMulti(self, reponames=None, logger=NoopLogger()):
     def StreamDeserializeMulti(self, processor, reponames=None, logger=NoopLogger()):
         deserializers = []
         for repo in self.repoman.GetRepositories(reponames):
-            deserializers.append(self.StreamDeserializer(self.__GetSerializedPath(repo)))
```

Comment:
Please remove unrelated changes: this

Reply:
Yep, knew these were unrelated thus put them in a seperate commit. :)
Couldnt have tested the solution and these changes were what stopped errors for me.